schema.sql

```diff
@@ -66,4 +66,6 @@ CREATE TABLE visits (
   FOREIGN KEY (vet_id) REFERENCES vets(id),
   FOREIGN KEY (animal_id) REFERENCES animals(id)
 );
+CREATE INDEX idx_email ON owners (email);
 
```

Comment:
- [ ] Kindly add the missing index to your `schema.sql` file. The file should contain three indexes instead of two.